sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -31,6 +33,15 @@ def streaming_test():
 def stream_compressed_header_error():
     yield b'test'
 
+def stream_compressed_no_header():
```

Comment:
I think there's already a sync version of this test, so I don't think you have to add a new endpoint in the testserver, can you double check? Thanks!